.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
.idea should be in gitignore. Please exclude this file from commit

Reply:
Done